python/sparknlp/annotator.py

```diff
@@ -99,7 +100,12 @@ def __init__(self):
         self._setDefault(
             targetPattern="\\S+",
             contextChars=[".", ",", ";", ":", "!", "?", "*", "-", "(", ")", "\"", "'"],
-            caseSensitiveExceptions=True
+            caseSensitiveExceptions=True,
+            exceptions=[],
+            infixPatterns=[],
+            prefixPattern="\\A([^\\s\\w\$\\.]*)",
+            splitChars=[],
+            suffixPattern="([^\\s\\w]?)([^\\s\\w]*)\\z')"
```

Comment:
These defaults do not match the defaults on the Scala side. Also empty is not equal to unset, which is a subtle but important difference. The code sometimes check if the user defined any value at all, rather than an empty content.

Reply:
Fair enough, I was trying to insert sane defaults based on the docstring.